package.json

```diff
@@ -21,13 +22,13 @@
     "eslint": "eslint packages/**",
     "clean": "rm -rf packages/**/lib packages/**/node_modules packages/**/*.tsbuildinfo package-lock.json node_modules dist && git checkout package-lock.json",
     "typecheck": "tsc --noEmit --skipLibCheck",
-    "compile": "npm run respect:parser:generate && tsc -b tsconfig.build.json && npm run build-docs:copy-assets",
+    "compile": "tsc -b tsconfig.build.json && npm run respect:parser:generate && npm run build-docs:copy-assets",
     "prepare": "npm run compile",
-    "cli": "ts-node packages/cli/src/index.ts",
+    "cli": "node --import tsx packages/cli/src/index.ts",
     "precli": "npm run compile",
     "release": "changeset publish",
     "pack:prepare": "./scripts/local-pack.sh",
-    "respect:parser:generate": "pegjs --format commonjs packages/respect-core/src/modules/runtime-expressions/abnf-parser.pegjs",
+    "respect:parser:generate": "pegjs --format commonjs --output packages/respect-core/lib/modules/runtime-expressions/abnf-parser.cjs packages/respect-core/src/modules/runtime-expressions/abnf-parser.pegjs && cp packages/respect-core/lib/modules/runtime-expressions/abnf-parser.cjs packages/respect-core/src/modules/runtime-expressions/abnf-parser.cjs",
```

Comment:
Store abnf-parser.cjs in the `lib/` folder (as we actually need it there) but copy it back in `src/` so Typescript doesn't fail.